racksdb/web/ui/index.html

```diff
@@ -4,6 +4,9 @@
     <meta charset="UTF-8">
     <link rel="icon" href="/favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
```

Comment:
The UI should not depend nor load on external files from CDN. The file is bundled in leaf rpm package:

```console
$ ls -lah node_modules/leaflet/dist
total 2,9M
drwxrwxr-x 3 remi remi 4,0K mars  12 18:58 .
drwxrwxr-x 4 remi remi 4,0K mars  12 18:58 ..
drwxrwxr-x 2 remi remi 4,0K mars  12 18:58 images
-rw-rw-r-- 1 remi remi  15K mars  12 18:58 leaflet.css
-rw-rw-r-- 1 remi remi 145K mars  12 18:58 leaflet.js
-rw-rw-r-- 1 remi remi 221K mars  12 18:58 leaflet.js.map
-rw-rw-r-- 1 remi remi 415K mars  12 18:58 leaflet-src.esm.js
-rw-rw-r-- 1 remi remi 846K mars  12 18:58 leaflet-src.esm.js.map
-rw-rw-r-- 1 remi remi 440K mars  12 18:58 leaflet-src.js
-rw-rw-r-- 1 remi remi 846K mars  12 18:58 leaflet-src.js.map
```

We should use this file instead!